logstash_async/cache.py

```diff
@@ -0,0 +1,51 @@
+import abc
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class Cache(object):
+
+    @abc.abstractmethod
+    def add_event(self, event):
+        """Add the event to the cache
```

Comment:
What do you think about adding a comment here stating that this is the method which is called from other threads while all other cache/database methods are called from the log processing worker thread.

But I can do as well later on.